### DISCUSSION TITLE
nhl support

### DISCUSSION THREAD

**Discussion by SwapnilShahGit** [mattdodge/yahoofantasy#49] [General]
Is there a reason nhl leagues are not supported?

If not, happy to help contribute to add functionality.

**Comment by SwapnilShahGit** [comment]
@mattdodge @WhosNickDoglio I just saw this issue: https://github.com/mattdodge/yahoofantasy/issues/6

I have a commit ready for NHL game codes, wondering if you can give me contributor access so I can open a PR.

**Comment by mattdodge** [comment]
@SwapnilShahGit you shouldn't need contributor access to send in a PR, check out [this GitHub guide](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) if you need a jump start there. Thanks for prepping the commit, would love to have NHL support!

**Comment by SwapnilShahGit** [comment]
Created a PR here: https://github.com/mattdodge/yahoofantasy/pull/50

Thanks in advance for having a look!

**Comment by TheyCallMeCheng** [comment]
Why and how am I mentioned here? Lol
